package.json

```diff
@@ -1,21 +1,39 @@
 {
-  "name": "root",
-  "private": true,
-  "devDependencies": {
-    "@types/jest": "^24.0.18",
-    "lerna": "^3.16.4",
-    "lodash.camelcase": "^4.3.0"
-  },
-  "scripts": {
-    "watch": "lerna run watch --parallel --stream",
-    "build": "lerna run build --stream",
-    "build:debug": "lerna run build:debug --stream",
-    "bootstrap": "lerna bootstrap",
-    "pretest": "lerna run build",
-    "test": "lerna run test --stream",
-    "lint": "lerna run lint --stream",
-    "lint:fix": "lerna run lint:fix --stream",
-    "release": "lerna bootstrap && npm run test && npm run build && lerna publish --force-publish=* --conventional-commits",
-    "prerelease": "lerna bootstrap && npm run test && npm run build && lerna publish --dist-tag prerelease --force-publish=* --preid rc"
-  }
+	"name": "root",
+	"private": true,
+	"devDependencies": {
+		"@types/jest": "^24.0.18",
+		"lerna": "^3.16.4",
+		"lodash.camelcase": "^4.3.0"
+	},
+	"dependencies": {
+		"@salte-auth/popup": "1.0.0-rc.2",
+		"@salte-auth/salte-auth": "3.0.0-rc.8",
+		"jest": "^24.9.0",
+		"jest-environment-jsdom-fifteen": "^1.0.0",
+		"jest-fetch-mock": "^2.1.2",
+		"mitt": "^1.1.3",
+		"rimraf": "^3.0.0",
+		"rollup": "^1.21.4",
+		"rollup-plugin-commonjs": "^10.1.0",
+		"rollup-plugin-node-resolve": "^5.2.0",
+		"rollup-plugin-sourcemaps": "^0.4.2",
+		"rollup-plugin-terser": "^5.1.2",
+		"rollup-plugin-typescript2": "^0.24.2",
+		"ts-jest": "^24.1.0",
+		"tslint": "^5.20.0",
+		"typescript": "^3.6.3"
+	},
+	"scripts": {
+		"watch": "lerna run watch --parallel --stream",
+		"build": "lerna run build --stream",
+		"build:debug": "lerna run build:debug --stream",
+		"bootstrap": "lerna bootstrap --hoist",
```

Comment:
This command is good, but I think we should add `postinstall` (like we have in Airship) so that we don't forget to lerna bootstrap.

Reply:
Yeah, that's better. Done on the next commit.